python/tests/test_aggregates.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+from feldera import SQLContext
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data = [{"insert":{"id": 0, "c1": 1, "c2": 2, "c3": 3, "c4": 4, "c5": 5, "c6": 6, "c7": None, "c8": 8}},]       
+        return super().setUp()
+        
+    def execute_query(self, expected_data, pipeline_name, table_name, view_name, view_query):
```

Comment:
let's remove the "view_name" and always use the same view name in all tests
this will work if each program is executed separately.